packages/popmotion-pose/src/dom/pose.ts

```diff
@@ -52,6 +53,7 @@ const createPoseConfig = (
       onDragEnd,
       onPressStart,
       onPressEnd,
+      onPoseComplete,
```

Comment:
is this fixing some wrongly passed prop?

Reply:
oh, you are passing it to the config, currently `onPoseComplete` is not part of the config - it's a prop so it can easily change over time and with your current implementation you'd call stale handlers